netstandard/ref/System.ComponentModel.cs

```diff
@@ -50,6 +50,7 @@ public sealed partial class AsyncOperation
         internal AsyncOperation() { }
         public System.Threading.SynchronizationContext SynchronizationContext { get { throw null; } }
         public object UserSuppliedState { get { throw null; } }
+        ~AsyncOperation() { }
```

Comment:
This should be implementation detail

Reply:
Given this type is sealed I agree we can hide it as an implementation detail, however if this type was unsealed we would need to expose it so the right thing can happen for derived types.

Reply:
Removed